proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -369,8 +374,8 @@ def build(self):
         if ((self.config.pressure_change_type != PressureChangeType.fixed_per_stage)
                 or (self.config.mass_transfer_coefficient == MassTransferCoefficient.calculated)):
             self.length = Var(
```

Comment:
I wrote this in such a way that length or width were only built when needed. Allow me to walk you through the logic as it stands now:
- In the baseline case, the user just fixes membrane area itself; i.e., there's no need to explicitly define length or width since neither are used anywhere else.
- the `eq_area` constraint is only active when the `mass_transfer_coefficient` config option is set to `MassTransferCoefficient.calculated`, and this is once case when length and width are both needed explicitly. 
- `width` is needed when either `self.config.mass_transfer_coefficient == MassTransferCoefficient.calculated` is set, `self.config.pressure_change_type == PressureChangeType.calculated` is set, or both are set. 
- `length` is needed when `width` is needed (previous bullet), but even if the options from the previous bullet were not active, `length` is also required when `self.config.pressure_change_type == PressureChangeType.fixed_per_unit_length`. In this case, `width` wouldn't be necessary. This is the exception.

That is why length and width were not built together.